File: src/main/java/com/lawencon/elearn/dto/quizcollection/QuizCollectionInsertResDto.java
package com.lawencon.elearn.dto.quizcollection;

public class QuizCollectionInsertResDto {
	private QuizCollectionInsertDataResDto data;
	private String message;
	public QuizCollectionInsertDataResDto getData() {
		return data;
	}
	public void setData(QuizCollectionInsertDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}

File: frontend/src/app/dto/quiz-collection/quiz-collection.insert.res.dto.ts
import { QuizCollectionInsertDataResDto } from "./quiz-collection.insert.data.res.dto";

export class QuizCollectionInsertResDto{
    private _data! : QuizCollectionInsertDataResDto
	private _message! : string

    public get data(): QuizCollectionInsertDataResDto {
        return this._data;
    }

    public set data(data: QuizCollectionInsertDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }

}